### ISSUE TITLE
Can you make this mysql agnostic from the start?

### ISSUE THREAD

**Issue by morenoh149** [bellsaladin/Keystone-Mysql#1]
Can you make this mysql agnostic from the start? I'm a big fan of postgres. I think keystone should support RDMS's and nosql solutions (sqlite, elasticsearch, etc.)

**Comment by bellsaladin** [comment]
I agree with you, it will be great if keystone supported RDBMS & noSQL databases but the differencies between these two database categories, if not also between database systems of the same category, makes it quite a challenge.
Though, the implementation i'm working on is based on Sequelize JS which supports multiple RDBMS such as MySql, Postgres and MySQLServer. 
So there is a good chance this project will be able extend support to Postgres & MSSQL.
